### PR TITLE
#831: Add touchScrollEnabled prop to Tablo component (closes #831)

### LINKED ISSUE #831: Add touchScrollEnabled prop to Tablo component
## requirements
To enable touch scroll for Tablo we need to add a `touchScrollEnabled` prop. 

## specs
{optional: describe technical specs to implement this feature, if not obvious}

## misc
http://schrodinger.github.io/fixed-data-table-2/api-table.html

### PR DESCRIPTION
**Pull request by ascariandrea**
Closes #831

### tests performed
- table scrolls with touch events ✅ 
- tap on desired position in scrollbars lane trigger a scroll in the desired position (just a little bit slow) ✅ 

Things to be noticed:
- dragging scrollbar cause a scroll in the opposite direction of touch gesture 

- [ ] Google Chrome
- [ ] Internet Explorer

> An example of partial test is a field displaying 4 possible values. If 3 values are tested, with screenshots, and 1 is not, then it should be mentioned here.

### CODE REVIEW

File: src/table/README.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ A table component based on fixed-data-table
 ## Props
 |Name|Type|Default|Description|
 |----|----|-------|-----------|
+| **touchScrollEnabled** | <code>Boolean</code> | <code>false</code> | *optional*. Enable touch scroll |
```
Comment (FrancescoCioria) [inline src/table/README.md]: `table` is actually the old (deprecated) `fixed-data-table` component, you should revert this edit

(this component will soon be removed from brc)

**Comment by FrancescoCioria** [comment]
@francescogior can you give a look to this tablo related PR? to me it looks fine (apart minor comment)

File: src/table/README.md (hunk 1 above)
Comment (ascariandrea) [inline src/table/README.md]: `Tablo` uses `Table` underneath :)

Comment (ascariandrea) [inline src/table/README.md]: No, my bad. The components have the same name!

**Pull request merged**